README.md

```diff
@@ -4,3 +4,4 @@ Recommended Changes in VaccineAPI Project for ProfileImage, SignatureImage to wo
             // dbDoctor.ProfileImage=doctorDTO.ProfileImage;
             // dbDoctor.SignatureImage=doctorDTO.SignatureImage;
 
```

Comment:
@abdulmoizdev 
what is the change in this file required for?
it nothing, revert the change and try remove it form PR